app/services/background.py

```
@@ -339,41 +344,162 @@ async def _autonomous_cycles_service(self):
+                # Get all users with active exchange accounts
+                from app.core.database import get_database
+                from app.models.exchange import ExchangeAccount
+                from sqlalchemy import select, and_, distinct
+                import json
+                
+                async for db in get_database():
+                    # Find all users with active exchange accounts
+                    stmt = select(distinct(ExchangeAccount.user_id)).where(
+                        and_(
+                            ExchangeAccount.status == "active",
+                            ExchangeAccount.trading_enabled == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `ExchangeAccount.trading_enabled:` for truth checks

Replace with `ExchangeAccount.trading_enabled`

(E712)